docs/source/_src_user_guide/state_propagation/propagation_setup/torque_models/available.rst

```diff
@@ -1,125 +1,79 @@
 
-Aerodynamic Torque
-######################
-
-Torque exerted by a body with an atmosphere model and shape model on another body.
-The settings are created as follows, for an aerodynamic torque exerted by "Earth" on the propagated body "Vehicle":
-
-.. tabs::
-
-     .. tab:: Python
 
-      .. toggle-header::
-         :header: Required **Show/Hide**
-
-      .. literalinclude:: /_src_snippets/simulation/propagation_setup/torque_models/aerodynamic_torque_example.py
-         :language: python
-
-     .. tab:: C++
+.. note::
+   In TudatPy, torque models are defined through factory functions, which define the properties required of
+   the torques, but do not perform any calculations themselves. These properties are stored through instances
+   of the :class:`~tudatpy.numerical_simulation.propagation_setup.torque.TorqueSettings` class or of its
+   derived classes.
 
-      .. code-block:: cpp
 
-            SelectedTorqueMap selectedTorqueModelMap;
-            selectedTorqueModelMap[ "Vehicle" ][ "Earth" ].push_back( std::make_shared< TorqueSettings >( aerodynamic_torque ) );
+.. contents:: List of available torque models
+   :depth: 1
+   :local:
 
+In certain pieces of code, such as when requesting the saving of a single torque (see :ref:`available_dependent_variables`
+for saving of dependent variables), you will need to supply an identifier for the type of torque you are requesting.
+See the list of supported identifier types in the API documentation:
+:class:`~tudatpy.numerical_simulation.propagation_setup.torque.AvailableTorque`.
 
-Requires the following environment models to be defined:
 
-* Atmospheric model for body exerting torque (see :ref:`environment_atmosphere_model`)
+Aerodynamic Torque
+######################
 
-* Shape model for body exerting torque (see :ref:`environment_shape_model`).
+| **Description**
+| The aerodynamic torque model can be created through the :func:`~tudatpy.numerical_simulation.propagation_setup.torque.aerodynamic` factory function.
 
-* Aerodynamic coefficient interface for body undergoing torque (see :ref:`environment_aerodynamic_coefficient_interface`).
+| **Dependencies**
+| 1. Atmosphere model for body exerting acceleration (see :ref:`environment_atmosphere_model`).
+| 2. Aerodynamic coefficient interface for body undergoing acceleration (see
+  :ref:`environment_aerodynamic_coefficient_interface`).
+| 3. Inertia tensor model for body undergoing acceleration.
+| 4. Current states of body undergoing acceleration and body with atmosphere.
 
 .. note::
 
     In the case that the aerodynamic coefficients are defined as a function of the vehicle orientation (e.g. angle of attack and sideslip angle), these angles can be manually or automatically defined.
 
-* Current state of bodies undergoing and exerting torque, either from an Ephemeris model (see :ref:`environment_ephemeris_model`) or from the numerical propagation.
-
+.. todo::
```

Comment:
@DominicDirkx could you please address this TODO?

Reply:
Thanks for identifying this missing point, I've pushed the change